src/stories/Header/component.js

```diff
@@ -0,0 +1,10 @@
+import styled from "styled-components";
+
+const Header = styled.header`
+    background-color: teal;
```

Comment:
We need to start considering consider to create a new file in your src folder, called something like theme.js where you define all of your styles (colours, paddings etc,) which you can import here and reuse.

That way you just need to change in one place.

Reply:
That was the plan, I just wanted to hard-code colour for the time being until I can import it from somewhere. "Theme" is a great idea, done!